### ISSUE TITLE
[object Object] written to log when looking up Lambda events

### ISSUE THREAD

**Issue by awschristou** [aws/aws-toolkit-vscode#782]
**Describe the bug**

When logging information about loading sample Lambda Events, the actual data is not being logged.

```
2019-10-11 17:00:34 [INFO]: building template...
2019-10-11 17:00:34 [INFO]: https://d3rrggjwfhwld2.cloudfront.net/LambdaSampleFunctions/SampleRequests/manifest.xml
2019-10-11 17:00:34 [INFO]: c:\.......\resources\vs-lambda-sample-request-manifest.xml
2019-10-11 17:00:36 [INFO]: querying manifest url
2019-10-11 17:01:53 [INFO]: [object Object]
2019-10-11 17:03:48 [INFO]: [object Object]
```

**To Reproduce**
1. Navigate to a Lambda Function in the AWS Explorer
2. Right click and choose Invoke
3. When the dialog opens, go into the Command Palette (View -> Command Palette) and select "View AWS Toolkit Logs"
4. see the log text with [object Object]

**Expected behavior**
* The data should be logged at debug level instead of info
* the object contents should be logged

**Desktop (please complete the following information):**
 - AWS Toolkit for Visual Studio Code Version: 1.1.0

**Comment by PseudoAj** [comment]
@awschristou I can take care of this. Feel free to assign this to me.

**Comment by awschristou** [comment]
Thank you @PseudoAj looking forward to your contribution!

**Comment by awschristou** [comment]
Hello @PseudoAj are you still interested in investigating this?

**Comment by PseudoAj** [comment]
@awschristou Yeah, I will try to make progress this week.

**Comment by awschristou** [comment]
Fixed and released in v1.6.0
